src/ape_cache/_cli.py

```diff
@@ -0,0 +1,40 @@
+import click
+import pandas as pd
+
+from ape.cli import NetworkBoundCommand, network_option
+from ape.utils import ManagerAccessMixin, logger  # type: ignore
+
+
+def get_engine():
+    return ManagerAccessMixin.query_manager.engines["cache"]
+
+
+@click.group(short_help="Query from caching database")
+def cli():
+    """
+    Command-line helper for managing query caching database.
+    """
+
+
+@cli.command(cls=NetworkBoundCommand, short_help="Initialize a new cache database")
+@network_option()
+def init(network):
+    get_engine().init_db()
+    logger.info("Caching database initialized.")
+
+
+@cli.command(
+    cls=NetworkBoundCommand, short_help="Call and print SQL statement to the cache database"
+)
+@network_option()
+@click.argument("sql")
+def query(sql, network):
```

Comment:
Is there a way to limit the sql to only READ operations?
Could someone use this command to delete the databases or inject misleading data in them?

Reply:
Yes, this is basically a shortcut to run whatever operations you want against the database. This cli endpoint should only ever be run locally.